src/list/manage/components/item-list-panel.js

```diff
@@ -35,9 +37,16 @@ export default function ItemListPanel({className, inputRef, totalItemCount,
       <PanelHeader className={styles.panelHeader}
                    border={hasItems ? "floating" : "none"}
                    toolbarClassName={styles.filterToolbar}>
-        <ItemFilter className={styles.itemFilter}
-                    inputRef={inputRef}/>
-        <AddItem/>
+        <div className={styles.firstRow}>
+          <ItemFilter className={styles.itemFilter}
+                      inputRef={inputRef}/>
+          <AddItem/>
+        </div>
+        <div className={styles.secondRow}>
+          <ListSort sort={sort} className={styles.listSort} {...props} />
```

Comment:
Are ListSort or ListCounter used anywhere else separately? Might reduce complexity a little to combine this whole row into a single component? (Well, you know, other than the busy work to combine them, changes tests, etc 😅 )

Reply:
Good questions! I think the ListCounter could abstract away the "foo entries found" code that's [currently inlined](https://github.com/mozilla-lockbox/lockbox-addon/blob/master/src/list/popup/components/item-list-panel.js#L72-L92) in the popup's ItemListPanel, though it'd be a bit of work to add a non-numeric default state to show the "recently used entries. click to search" text when there's no filter.